src/Stripe.net/Services/PaymentIntents/PaymentIntentTransferDataOptions.cs

```diff
@@ -1,14 +1,16 @@
 namespace Stripe
 {
+    using System;
     using Newtonsoft.Json;
 
     public class PaymentIntentTransferDataOptions : INestedOptions
     {
-        [JsonProperty("amount")]
-        public long? Amount { get; set; }
-
         // This is only available on creation and not update
         [JsonProperty("destination")]
         public string Destination { get; set; }
+
+        [Obsolete("Use ApplicationFeeAmount instead")]
```

Comment:
Same here, can you change the message to `"Use PaymentIntent*Options.ApplicationFeeAmount"`?